PWA/src/components/Timesheet/ConfirmSubmission.vue

```diff
@@ -157,16 +150,139 @@
         //Data to be submitted
         submitData: null,
 
+        // All the errors of this form
+        errors: [],
+        isValid: this.valid,
+
         //URL for the AppServer
         url: process.env.VUE_APP_SERVER_URL.concat("Submit"),
       };
     },
 
+    watch: {
+      valid(newVal) {
+        this.isValid = newVal;
+      },
+    },
+
     methods: {
+      // Compute the sum of all serviceDeliveredOn totalHours with the totalHours field
```

Comment:
we want the total hours to be auto calculated in the totalhours column and the total hours field beneath the table.